sdk/digitaltwins/Azure.DigitalTwins.Core/README.md

```diff
@@ -135,7 +135,7 @@ For more information see the Code of Conduct FAQ or contact opencode@microsoft.c
 [digital_twins_client]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs
 [digital_twins_documentation]: https://docs.microsoft.com/azure/digital-twins/
 [azure_cli]: https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest
-[iot_cli_extension]: https://docs.microsoft.com/azure/iot-pnp/howto-use-iot-pnp-cli
+[iot_cli_extension]: https://github.com/Azure/azure-iot-cli-extension/releases
```

Comment:
Earlier link points to Device model repository concepts

Reply:
We can certainly add additional link pointing to readme, here however I would like to see download link.